src/test/java/com/ddudu/todo/controller/TodoControllerTest.java

```diff
@@ -48,6 +56,52 @@ class TodoControllerTest {
   @Autowired
   ObjectMapper objectMapper;
 
+  @Nested
+  class POST_할_일_생성_API_테스트 {
+
+    String name;
+    LocalDateTime beginAt;
+
+    @BeforeEach
+    void setUp() {
+      name = faker.lorem()
+          .word();
+      beginAt = faker.date()
+          .birthday()
+          .toLocalDateTime();
+    }
+
+    @Test
+    void 할_일_생성을_성공한다() throws Exception {
+      // given
+      Long goalId = 1L;
+      CreateTodoRequest request = new CreateTodoRequest(goalId, name, beginAt);
+      TodoInfo response = TodoInfo.builder()
+          .id(1L)
+          .name(name)
+          .status(TodoStatus.UNCOMPLETED)
+          .build();
```

Comment:
```suggestion
      TodoInfo response = createTodoResponse(name);
```
`createTodoInfo()` 메서드가 아래 존재하는데 활용해보는 건 어떨까요? 해당 메서드에 name 추가해도 좋을 것 같습니다..! (저도 우선 임시로 만들어 놓은거라 마음껏 활용하셔도 됩니다!)

Reply:
헉 제가 실수로 놓쳤나보네요! 수정해 놓겠습니당~~